pangram/pangram.rb

```diff
@@ -0,0 +1,22 @@
+class Pangram
+   def self.pangram?(phrase)
+     if phrase == ''
+       return false
+     end
+
+     numOfChar = 0
+     phrase = phrase.upcase
+     phrase = phrase.split('')
+
+     ('A'..'Z').each do|index|
+       if (phrase.include?index)
+        numOfChar += 1
+       end
+    end
+     if(numOfChar == 26)
+       return true
+     else
+       return false
+     end
```

Comment:
@AjitThakor-btc This line of code can be merged into a single line like below...

`numOfChar.eql?(26)`

No need to write the return statement at the end of the line.